src/components/TodoList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ReadyTodos } from '../react-app-env';
+
+type Props = {
+  readyTodos: ReadyTodos[];
+};
+export const TodoList: React.FC<Props> = ({ readyTodos }) => (
+  <div className="container">
+    {readyTodos.map(todo => (
+      <div key={todo.id}>
```

Comment:
I think better to use React Fragment here, because this div is unnecessary

Reply:
With this div I stylize my columns. 

Without div : http://joxi.ru/8An45XDUNP1B1m